lib/logstash/inputs/jdbc.rb

```diff
@@ -160,6 +160,14 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to force the lowercasing of identifier fields
   config :lowercase_column_names, :validate => :boolean, :default => true
 
+  # Default encoding applied to all columns unless no specific encoding is defined
+  # in the columns_to_encode variable. By default this value is not defined
+  config :default_encoding, :validate => :string
+
+  # It let's you select which string columns should be converted to UTF-8 from a given encoding type.
+  # To especify a column to be converted you can add the pair "column_name" => "original_encoding_type" to this hash.
+  config :columns_to_encode, :validate => :hash, :default => {}
```

Comment:
I suggest `:columns_encoding` instead of  `:columns_to_encode` - it feels more consistent